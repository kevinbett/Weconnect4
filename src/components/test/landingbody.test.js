import Home from "../LandingBody/index"
import { shallow } from "enzyme"
import { Divider } from "../../../node_modules/semantic-ui-react";

describe('landig body',() => { 
    it ('renders the lading body', () => { 
        expect(shallow(<Home/>).find("div.jumbotron").exists()).toBe(true);
    })
} )
