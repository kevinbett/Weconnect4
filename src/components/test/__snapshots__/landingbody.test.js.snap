// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`landig body renders the lading body 1`] = `
<div
  className="jumbotron"
>
  <h1
    class="display-4"
  >
    Welcome to WeConnect!
  </h1>
  <p
    class="lead"
  >
    Your business needs are now one click away...
  </p>
  <hr
    class="my-4"
  />
  <p
    class="lead"
  >
    <Link
      class="btn btn-primary btn-lg"
      replace={false}
      role="button"
      to="/register"
    >
      SignUp
    </Link>
             
    <Link
      class="btn btn-primary btn-lg"
      replace={false}
      role="button"
      to="/login"
    >
        Login  
    </Link>
  </p>
</div>
`;
